src/features/visual-querying/VisualQuerying.tsx

```diff
@@ -52,13 +52,16 @@ export function VisualQuerying(): JSX.Element {
       : null;
 
     // Send the query
-    void trigger({
-      name: name ?? undefined,
-      dateOfBirthStart: dateOfBirth ? dateOfBirth[0] : undefined,
-      dateOfBirthEnd: dateOfBirth ? dateOfBirth[1] : undefined,
-      dateOfDeathStart: dateOfDeath ? dateOfDeath[0] : undefined,
-      dateOfDeathEnd: dateOfDeath ? dateOfDeath[1] : undefined,
-    });
+    void trigger(
+      {
+        q: name ?? undefined,
+        dateOfBirthStart: dateOfBirth ? dateOfBirth[0] : undefined,
+        dateOfBirthEnd: dateOfBirth ? dateOfBirth[1] : undefined,
+        dateOfDeathStart: dateOfDeath ? dateOfDeath[0] : undefined,
+        dateOfDeathEnd: dateOfDeath ? dateOfDeath[1] : undefined,
+      },
+      true,
```

Comment:
What does this parameter do?

Reply:
Nevermind, I figured it out :)

Reply:
for documentation:

"the `trigger` function takes a second argument of `preferCacheValue?: boolean` in the event you want to skip making a request if cached data already exists."

see https://redux-toolkit.js.org/rtk-query/usage/queries#hook-types